distribution/docs/src/main/resources/content/_integrating/_endpoints/csw-endpoint.adoc

```diff
@@ -2020,89 +1016,47 @@ The following example shows how the newly inserted record could be updated to mo
+|`<csw:totalUpdated>0</csw:totalUpdated>`
+|No records were updated. Verify metacard id or search parameters.
+|===
 
-.Sample XML Transaction Delete Response
-[source,xml]
-----
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<csw:TransactionResponse
-                         xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
-                         xmlns:ns10="http://www.w3.org/2001/XMLSchema-instance"
-                         ns10:schemaLocation="http://www.opengis.net/csw /ogc/csw/2.0.2/CSW-publication.xsd"
-                         version="2.0.2">
-    <csw:TransactionSummary>
-        <csw:totalInserted>0</csw:totalInserted>
-        <csw:totalUpdated>0</csw:totalUpdated>
-        <csw:totalDeleted>1</csw:totalDeleted>
-    </csw:TransactionSummary>
-</csw:TransactionResponse>
-----
+=== CSW Endpoint Publication/Subscription Examples
 
-==== Subscription `GetRecords` Operation
+The subscription `GetRecords` operation is very similar to the `GetRecords` operation used to search the catalog
+but it subscribes to a search and sends events to a `ResponseHandler` endpoint as metacards are ingested matching
+the `GetRecords` request used in the subscription.
+The `ResponseHandler` must use the https protocol and receive a HEAD request to poll for availability and
+POST/PUT/DELETE requests for creation, updates, and deletions.
+The response to a `GetRecords` request on the subscription url will be an acknowledgement containing the original
+`GetRecords` request and a `requestId`.
+The client will be assigned a `requestId` (URN).
 
-The subscription `GetRecords` operation is very similar to the `GetRecords` operation used to search the catalog but it subscribes to a search and sends events to a `ResponseHandler` endpoint as metacards are ingested matching the GetRecords request used in the subscription.
-The `ResponseHandler` must use the https protocol and receive a HEAD request to poll for availability and POST/PUT/DELETE requests for creation, updates, and deletions.
-The response to a `GetRecords` request on the subscription url will be an acknowledgement containing the original GetRecords request and a requestId
-The client will be assigned a requestId (URN).
 A Subscription listens for events from federated sources if the `DistributedSearch` element is present and the catalog is a member of a federation.
 
-==== Subscription `GetRecords` HTTP GET
+.Adding a Subscription
+Send a `POST` request to the CSW endpoint.
 
-.`GetRecords` KVP (Key-Value Pairs) Encoding
+[source,https]
 ----
-${secure_url}/services/csw/subscription?service=CSW&version=2.0.2&request=GetRecords&outputFormat=application/xml&outputSchema=http://www.opengis.net/cat/csw/2.0.2&NAMESPACE=xmlns(csw=http://www.opengis.net/cat/csw/2.0.2)&resultType=results&typeNames=csw:Record&ElementSetName=brief&ResponseHandler=https%3A%2F%2Fsome.ddf%2Fservices%2Fcsw%2Fsubscription%2Fevent&ConstraintLanguage=CQL_TEXT&constraint=Text Like '%25'
+https://<FQDN>:<PORT>/services/csw
```

Comment:
❓ Should this be "https://<FQDN>:<PORT>/services/csw/**subscription**" instead?

Reply:
fixed